keyboards/ai03/equinox/rev0/rules.mk

```diff
@@ -0,0 +1 @@
+# Dummy rules.mk, rev1 uses parent rules.mk as is
```

Comment:
If you want to avoid driving a pin, you could add `BACKLIGHT_DRIVER = custom` where the default implementation is basically empty stub functions.

Reply:
There are only a handful of these prototype PCBs which have two columns not connected to I/O pins and this is one of those pins... this is as much of a hack as the bodge wires required to get it working 😜 Really don't want to add more code just for this.

Reply:
Well in theory its 1 rules.mk line instead of the 2 header lines, so not really more code.